pkg/rancher-desktop/assets/specs/command-api.yaml

```diff
@@ -208,6 +208,47 @@ paths:
         '400':
           description: An error occurred
 
+  /v1/port_forwarding:
```

Comment:
⏰

For a follow-up: list current port forwardings. (It makes sense to not do that in this PR.)

Reply:
I originally planned to support this, but it seems the Kubernetes API doesn't support it so it would need to be tracked via RD.

Reply:
Yeah, all of this forwarding is a RD-level thing [implemented here](https://github.com/rancher-sandbox/rancher-desktop/blob/4308edf6cf682afac63934ce5ba03c9aa3009979/pkg/rancher-desktop/backend/kube/client.ts#L75), so there's no Kubernetes API for it.  It's the equivalent of manually running `kubectl port-forward` a lot.

Reply:
Created as #6696